src/shared/utils/challenge-listing/sort.js

```diff
@@ -6,25 +6,25 @@ import moment from 'moment';
 import { sumBy } from 'lodash';
 
 export const SORTS = {
-  // CURRENT_PHASE: 'current-phase',
+  CURRENT_PHASE: 'current-phase',
   MOST_RECENT: 'updated',
   MOST_RECENT_START_DATE: 'startDate',
   // NUM_REGISTRANTS: 'num-registrants',
   // NUM_SUBMISSIONS: 'num-submissions',
-  // PRIZE_HIGH_TO_LOW: 'prize-high-to-low',
-  // TIME_TO_REGISTER: 'time-to-register',
-  // TIME_TO_SUBMIT: 'time-to-submit',
+  PRIZE_HIGH_TO_LOW: 'prizeSets.prizes.value',
```

Comment:
@PrakashDurlabhji Please change `prizeSets.prizes.value` to `overview.totalPrizes`